package.json

```diff
@@ -164,5 +165,33 @@
   "license": "BSD-3-Clause",
   "engines": {
     "node": ">=12"
+  },
+  "exports": {
+    "./css": "./dist/answers.css",
+    "./rtl-css": "./dist/answers.rtl.css",
+    ".": "./dist/answers-umd.js",
+    "./esm": "./dist/answers-modern.js",
+    "./esm.min": "./dist/answers-modern.min.js",
+    "./template": "./dist/answerstemplates.compiled.min.js",
+    "./es": "./dist/es-answers-umd.js",
+    "./es/esm": "./dist/es-answers-modern.js",
+    "./es/esm.min": "./dist/es-answers-modern.min.js",
+    "./es/template": "./dist/es-answerstemplates.compiled.min.js",
+    "./fr": "./dist/fr-answers-umd.js",
+    "./fr/esm": "./dist/fr-answers-modern.js",
+    "./fr/esm.min": "./dist/fr-answers-modern.min.js",
+    "./fr/template": "./dist/fr-answerstemplates.compiled.min.js",
```

Comment:
What do the import paths look like with these exports? Would it be possible to just rely on the natural filepath from the dist folder instead of having a custom path for every asset?

Reply:
It's possible to use the natural filepaths. After the SDK multilang bundle investigation meeting, it was decided we should provide custom entry points to make it easier to access the multilang bundles

Reply:
ah got it, would it be easier to just reorganize the dist folder before an npm publish? Also what is the "exports" field? I couldn't find it in https://docs.npmjs.com/cli/v7/configuring-npm/package-json#exports 
I do see it [here](https://nodejs.org/api/packages.html#packages_exports) though, would people need to be on a newer version of node/npm to use exports?

Reply:
hmm you mean like reorganize dist folder for different languages without the custom entry points? 
Yea, from Node.js 12+, otherwise it just ignores it and use `main` field as normal

Reply:
@tmeyer2115 what's your thought on just reorganizing the dist folder?
I can add test to check if it generates the expected export field. Export field would need update if we ever add new languages to SDK (like arabic soon). Probably just in prepublishOnly should be ok?

Reply:
oh just to clarify I'm cool with having this exports field in the package.json! Was just throwing out an alternative and wondering if it would stick or not.

Reply:
That's an interesting idea. On NPM I don't think we need the source code, so maybe it makes sense to not even have a dist folder on NPM and we instead move everything from dist into the top level